zerver/webhooks/ansibletower/view.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Dict, Iterable, Optional, List
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile
+
+import ujson
+import operator
+
+ANSIBLETOWER_SUBJECT_TEMPLATE = "{name}"
```

Comment:
This is probably redundant, you can just set `subject` to whatever contains `name` directly. There is no need for a template here. :)